api/docs/disperser.md

```diff
@@ -315,7 +315,17 @@ RetrieveBlobRequest contains parameters to retrieve the blob.
 <a name="disperser-BlobStatus"></a>
 
 ### BlobStatus
-
+BlobStatus represents the status of a blob.
+The status of a blob is updated as the blob is processed by the disperser.
+The status of a blob can be queried by the client using the GetBlobStatus API.
+Intermediate states are states that the blob can be in while being processed, and it can be updated to a differet state:
+- PROCESSING
+- DISPERSING
```

Comment:
`DISPERSING` state is hidden to users. It surfaces up as `PROCESSING` in this state

Reply:
needs to fix in .proto